.github/workflows/project-build-test.yaml

```diff
@@ -17,6 +17,8 @@ on:
   push:
     branches:
       - main
+    tags:
+      - 'v*'
```

Comment:
Do we want to have this run on all tags? Or will we just set the convention that all tags start with "v", even if they're non-numeric?

Reply:
I was thinking we just run on versioned tags, but we could change that